### PR TITLE
Simplify local settings file

### PR DESCRIPTION
**Pull request by techgique**
This was the result of conversation about simplifying the changelog's migration notes for recent logging settings changes. We discussed how it would be simpler to instruct people to just re-copy the example file and how it was a good opportunity to simplify the local settings file. This is my work from a thorough review of the related settings and documentation. I hope these changes will make standing up and configuring ONI simpler to follow and dive deeper if people want.

Note this will require replacing/updating your `settings_local.py` file for testing or your app will probably be horribly broken :) Relatedly, I think our next release should be a minor version due to this migration requirement. Would like all eyes on this one, @AndrewGearhart, @jduss4, @jechols and you'll definitely want to look through commit-by-commit or the diff will be unintelligible. :bow: 

## Submitter Checklist

- [X] Verify all tests pass
  - Run tests with `docker-compose -f test-compose.yml -p onitest up test`
- [X] Update `README.md` and `docs/` as necessary
  - [X] If a `manage.py` command is added, deleted, or modified its help text must
    be valid and it should be documented in detail in
    `docs/advanced/admin-commands.md`
- Update `CHANGELOG.md`
  - [X] Describe change(s) in appropriate section(s)
  - [X] List self in Contributors section
- If a release PR:
  - [X] In `CHANGELOG.md`, replace `[Unreleased]` with version and update compare link
  - [X] Update `core/version.py` with new version
- [X] Resolve merge conflicts
- [X] @mention individual(s) you would like to review the PR
  - Reviews for releases must come from a reviewer at another institution

## Reviewer Checklist

- [ ] Verify all tests pass
- [ ] Review changes for behavior, bugs, and compliance with [Development
  Standards](https://github.com/open-oni/open-oni/tree/dev/CONTRIBUTING.md#development-standards)
  - Request the submitter make any changes rather than pushing changes yourself.
    Otherwise ask another reviewer to look at any changes you have made.
- [ ] If a release, follow the [Release
  Checklist](https://github.com/open-oni/open-oni/tree/dev/CONTRIBUTING.md#release-checklist)

### CODE REVIEW

File: onisite/test_settings.py [hunk 1]
```diff
@@ -41,3 +41,13 @@
...
+
+# Reset back to Django defaults to keep test output simple
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+    }
+}
+
+del LOGGING
```
Comment (techgique) [inline onisite/test_settings.py:55]: Tangential question for a possible new issue: If this line is removed there are some warnings and errors that come through in the testing logging. Should our tests give cleaner output if logging is enabled beyond out of the box Django logging?

File: onisite/settings_local_example.py [hunk 2]
```diff
@@ -179,62 +93,3 @@
 """
 TOO_BUSY_LOAD_AVERAGE = 64
```
Comment (jechols) [inline onisite/settings_local_example.py:95]: Unnecessary newline here (git diff highlights it for me, though github doesn't)

Comment (jechols) [reply onisite/settings_local_example.py:95]: Oops, wrong file.  This is unnecessary too but you didn't add it.

File: onisite/settings_base.py [hunk 3]
```diff
@@ -120,3 +124,149 @@
...
+        'django.middleware.common.CommonMiddleware',
+        'django.middleware.csrf.CsrfViewMiddleware',
+        'django.contrib.messages.middleware.MessageMiddleware',
+        'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    )
+
+    # Fingerprint compiled static files with MD5 hash of contents
+    # Store hashes in STATIC_ROOT directory as staticfiles.json
+    STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
+
```
Comment (jechols) [inline onisite/settings_base.py:272]: Unnecessary newline here

File: CHANGELOG.md [hunk 4]
```diff
@@ -35,34 +35,49 @@ Markdown Spec](https://github.github.com/gfm/).
...
-- Logging now works as expected
+- Logging to `log/debug.log` file in addition to the console
 - Pinned docker-compose users to RAIS 3.x to avoid breakages when 4.x ships
 
 ### Added
- - Enabled apache mod_ssl in web image build
- - Included codebase as part of Dockerfile build
+- Enabled apache mod_ssl in web image build
+- Included codebase as part of Dockerfile build
+- `ONI_STORAGE_PATH` env var, missed in previous edits to `docker-compose.yml`
```
Comment (jechols) [inline CHANGELOG.md:44]: This is something that may be useful for bare-metal installs, but we *don't* want people to have the option to change it in the docker-compose setup unless they **really** know what they're doing.  Without properly changing mount points, altering `ONI_STORAGE_PATH` will make ONI stop functioning.

Comment (techgique) [reply CHANGELOG.md:44]: Removed. I discussed this and the broader concepts with Andrew and I think he gave me a better grasp of why this is bad practice in general for Docker. I have other ideas for cleaning up things with Docker env vars and some of our bash scripting, but I'll make a separate PR for those to not hold this up :+1:

**Review by jechols — DISMISSED** [review]

**Comment by jechols** [comment]
FYI there was some chatter from me in slack about tests not passing.  If you skim those, you might think this is a busted PR, but it is not.  It's a problem from my `ENV` directory, which I have a branch up to fix.

File: onisite/test_settings.py (hunk 1 above)
Comment (techgique) [reply onisite/test_settings.py:55]: Created an issue for this: https://github.com/open-oni/open-oni/issues/567

**Review by jechols — APPROVED** [review]
Looks good!

**Pull request merged**